nemo/collections/llm/gpt/data/packed_sequence.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+
+from nemo.collections.common.tokenizers import TokenizerSpec
+from nemo.collections.llm.gpt.data.core import create_sft_dataset
+from nemo.utils import logging
+from nemo.utils.sequence_packing_utils import create_hist, create_packing_strategy, fill_packing_strategy
+
+
+def tokenize_dataset(path: Path, tokenizer: TokenizerSpec, max_seq_length: int, seed: int):
```

Comment:
add docstring for this file and functions

Reply:
thanks, done!